### ISSUE TITLE
About the split of ABCT dataset

### ISSUE THREAD

**Issue by WuGuangHeng** [ZAX130/ModeTv2#2]
Thanks for your work. I have some questions about the  Abdomen CT (ABCT) dataset. Does this dataset derive from the cross-modality dataset Abdomen MR-CT in L2R? I found that I can't figure out the split of the dataset (train/test: 35/8). 
![image](https://github.com/ZAX130/ModeTv2/assets/78968948/f0d09fc7-8ace-4a97-a487-7b254fdf1495)
I am wondering the exact way of your split. Hope for your reply. Thanks!

**Comment by ZAX130** [comment]
I am very sorry for your misunderstanding, this may be an oversight in my paper writing. I used the remaining data as a validation set, but forgot to mention the validation set in the paper.

**Comment by Pull2MAX** [comment]
> I am very sorry for your misunderstanding, this may be an oversight in my paper writing. I used the remaining data as a validation set, but forgot to mention the validation set in the paper.非常抱歉造成了您的误解，这可能是我在论文撰写中的疏忽。我使用了剩余数据作为验证集，但忘记在论文中提及验证集。

作者应该用的是l2r里面的ct->ct吧，然后选了四个label？

**Comment by ZAX130** [comment]
腹部MR-CT的CT



---Original---
From: ***@***.***&gt;
Date: Mon, Jan 6, 2025 16:22 PM
To: ***@***.***&gt;;
Cc: "Haiqiao ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [ZAX130/ModeTv2] About the split of ABCT dataset (Issue #2)




  
I am very sorry for your misunderstanding, this may be an oversight in my paper writing. I used the remaining data as a validation set, but forgot to mention the validation set in the paper.非常抱歉造成了您的误解，这可能是我在论文撰写中的疏忽。我使用了剩余数据作为验证集，但忘记在论文中提及验证集。
  
作者应该用的是l2r里面的ct-&gt;ct吧，然后选了四个label？

**Comment by Pull2MAX** [comment]
> 腹部MR-CT的CT

# 桥哥，我仔细看了下l2r竞赛数据，还是对你筛选数据的逻辑有点疑问。如果方便回答的话可以回答我的问题吗？
1. 我不太懂为什么只用mr->ct里面的ct，既然是做的是ct->ct，为啥不直接选Abdomen CTCT 的数据集呢？这个数据集里面直接就有了30个label，那么直接用Abdomen CT -->CT两两配准行吗？
2. Abdomen MRCT 数据集中，训练集：57个CT图像，测试集：8个CT图像，为什么你在论文中说选35测试呢，我不知道是怎么筛选的，还有22个是怎么落选的？

![image](https://github.com/user-attachments/assets/d7ad05bb-242b-4287-9024-4f24c24fa55c)

**Comment by ZAX130** [comment]
mr ct里的ct数据集里数据量大一些，并且有mask方便居中裁剪统一对齐。而且mr-ct我实际上只用了L2R_Task1_CT文件夹里的数据。



---Original---
From: ***@***.***&gt;
Date: Mon, Jan 6, 2025 21:49 PM
To: ***@***.***&gt;;
Cc: "Haiqiao ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [ZAX130/ModeTv2] About the split of ABCT dataset (Issue #2)




  
腹部MR-CT的CT
  
桥哥，我仔细看了下l2r竞赛数据，还是对你筛选数据的逻辑有点疑问。如果方便回答的话可以回答我的问题吗？
  
我不太懂为什么只用mr-&gt;ct里面的ct，既然是做的是ct-&gt;ct，为啥不直接选Abdomen CTCT 的数据集呢？这个数据集里面直接就有了30个label，那么直接用Abdomen CT --&gt;CT两两配准行吗？
 
Abdomen MRCT 数据集中，训练集：57个CT图像，测试集：8个CT图像，为什么你在论文中说选35测试呢，我不知道是怎么筛选的，还有22个是怎么落选的？
  
image.png (view on web)

**Comment by Pull2MAX** [comment]
感谢🙏，很有帮助，我再去研究下，您的 baseline 都写的特别好，大道至简👍



---Original---
From: "Haiqiao ***@***.***&gt;
Date: Mon, Jan 6, 2025 22:18 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ZAX130/ModeTv2] About the split of ABCT dataset (Issue #2)




 mr ct里的ct数据集里数据量大一些，并且有mask方便居中裁剪统一对齐。而且mr-ct我实际上只用了L2R_Task1_CT文件夹里的数据。 
 
 
 
 ---Original--- 
 From: ***@***.***&amp;gt; 
 Date: Mon, Jan 6, 2025 21:49 PM 
 To: ***@***.***&amp;gt;; 
 Cc: "Haiqiao ***@***.***&amp;gt;;"State ***@***.***&amp;gt;; 
 Subject: Re: [ZAX130/ModeTv2] About the split of ABCT dataset (Issue #2) 
 
 
 
 
    
 腹部MR-CT的CT 
    
 桥哥，我仔细看了下l2r竞赛数据，还是对你筛选数据的逻辑有点疑问。如果方便回答的话可以回答我的问题吗？ 
    
 我不太懂为什么只用mr-&amp;gt;ct里面的ct，既然是做的是ct-&amp;gt;ct，为啥不直接选Abdomen CTCT 的数据集呢？这个数据集里面直接就有了30个label，那么直接用Abdomen CT --&amp;gt;CT两两配准行吗？ 
   
 Abdomen MRCT 数据集中，训练集：57个CT图像，测试集：8个CT图像，为什么你在论文中说选35测试呢，我不知道是怎么筛选的，还有22个是怎么落选的？ 
    
 image.png (view on web)

**Comment by cyl0000** [comment]
所以想问一下您在论文中的35个训练集具体是用了哪些呢，我现在还是有些困惑是采用了L2R_Task1_CT中的数据还是L2R_Task2中的数据，只是用了Task1的索引，不知道您是否方便提供一个35个数据集的json列表
